NETEZZA.md

```diff
@@ -1,46 +1,24 @@
 #### Installing and testing with Netezza emulator
 
-The Netezza emulator is only available for windows installation. 
-It consists of 2 VMware images which theoretically could be run on any VMware environment, but this is not tested.
+The Netezza emulator VM is available as an Open Virtualisation archive (OVA) format file.
+It consists of a Red Hat Linux Netezza host image that contains a nested VM for the Netezza SPU.
+Because of the nested VM virtualisation of Intel VT-x/EPT or AMD-V/RVI is required. This feature is not currently availale with Oracle VirtualBox hence it is recommended to run the Netezza emulator with VMWare Player/Workstation.
 
 1. Install the Netezza Emulator from IBM Netezza Developer Network
    * Download [VMware](https://my.vmware.com/web/vmware/downloads) Player.
-   * Download the [Netezza Emulator](https://www14.software.ibm.com/webapp/iwm/web/reg/pick.do?source=swg-im-ibmndn&lang=en_US)
-   * Also download the Netezza Client components from the same location. This contains the JDBC driver and the IBM netezza Administrator GUI
-  
+   * Download the 7.2 version of the [Netezza Emulator](https://www14.software.ibm.com/webapp/iwm/web/reg/pick.do?source=swg-im-ibmndn&lang=en_US)
      **NOTE:** you will need to register and create an IBM developer network account
      
    * Install VWware 
-   * Verify the Netezza Emulator requirements and install with the installer
+   * Open the downloaded Netezza VM `.ova` file in VMWare
      **NOTE:** The Netezza Emulator typically requires >= 4G of ram
-
-   * Install the JDBC driver. This creates a seperate directory which contains the nzjdbc.jar needed.
-   * Optionally install the Netezza Administrator GUI (makes life a lot easier)
-     **NOTE:** the default passwords for the emulator are nz/nz for the linux host and admin/password for the database connection
+     **NOTE:** the default passwords for the emulator are `nz`/`nz` for the Linux host and `admin`/`password` for the database connection
```

Comment:
I think these NOTEs are rendered on a single line (actually in the the same paragraph where the last bullet started).